apps/central/src/components/Admin.vue

```diff
@@ -0,0 +1,137 @@
+<template>
+  <div>
+    <h1>Admin tools</h1>
+    <Spinner v-if="loading"></Spinner>
+    <MessageError v-else-if="error">{{ error }}</MessageError>
+    <div v-else>
+      <h2>Create/alter user</h2>
+      <MessageSuccess v-if="alterSuccess">{{ alterSuccess }}</MessageSuccess>
+      <MessageError v-if="alterError">{{ alterError }}</MessageError>
+      <Spinner v-if="alterLoading"></Spinner>
+      <form v-else class="form-inline bg-white">
+        <InputString
+          label="username"
+          v-model="username"
+          placeholder="Enter username"
+        />
+        <InputPassword
+          label="password"
+          v-model="password"
+          placeholder="Enter password"
+        />
+        <ButtonAction @click="alterUser">Update user</ButtonAction>
```

Comment:
When i create a new user the user is not added to the list until after page reload